driven/storage/adapter.go

```diff
@@ -452,6 +453,7 @@ func (sa *Adapter) CreateGroup(clientID string, current *model.User, group *mode
 				castedMemberships = append(castedMemberships, membership)
 			}
 		} else if current != nil {
+			externalIDs := map[string]string{"netID": current.NetID, "externalID": current.ExternalID}
```

Comment:
Please use under scores instead of camel names. Camel names are not wrong approach. The idea is just to keep common standard within the project.

Reply:
Also in this way of thoughts externalID is better to be renamed to uin, uiuc_uin if we gonna support more such attributes and external ids...

CC @shurwit